web_decimal_numpad_dot/static/src/js/numpad_dot.js

```diff
@@ -0,0 +1,75 @@
+/* License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_decimal_numpad_dot.FieldFloat', function (require) {
+"use strict";
+
+    var core = require('web.core');
+    var form_widgets = require('web.form_widgets');
+    var translation = require('web.translation');
+
+    var FieldFloatDecimalDot = form_widgets.FieldFloat.extend({
+        events: {
+            'keypress': function (e) {
```

Comment:
I can see that [`FieldFloat` extends `FieldChar`](https://github.com/odoo/odoo/blob/cd1a8f625276e3c40b320df73e4041004c77aea8/addons/web/static/src/js/views/form_widgets.js#L366), which [already defines an `events` property](https://github.com/odoo/odoo/blob/cd1a8f625276e3c40b320df73e4041004c77aea8/addons/web/static/src/js/views/form_widgets.js#L117-L121).

Most likely you are overriding that property here, instead of adding a new value to it (which is what you want to do, right?).

Please check it out; if it is like I say, you'll have to declare it this way:

```js
form_widgets.FieldFloat.include({
  init: function () {
    this.events.keypress = function (event) {...};
    return this._super.apply(this, arguments);
  },
});
```